<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>KeyPad</class>
 <widget class="QDialog" name="KeyPad" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Enter your pin!</string>
  </property>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>520</x>
     <y>20</y>
     <width>146</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Sans Serif</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Entered Number&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>280</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Sans Serif</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Message from User Interface&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP1" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>130</y>
     <width>148</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Serif</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP2" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>168</x>
     <y>130</y>
     <width>148</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Serif</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP3" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>326</x>
     <y>130</y>
     <width>149</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Serif</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP4" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>484</x>
     <y>130</y>
     <width>148</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Serif</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP5" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>642</x>
     <y>130</y>
     <width>148</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Serif</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP6" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>270</y>
     <width>148</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Serif</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP7" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>168</x>
     <y>270</y>
     <width>148</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Serif</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP8" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>326</x>
     <y>270</y>
     <width>148</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Serif</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP9" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>484</x>
     <y>270</y>
     <width>148</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Serif</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP0" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>642</x>
     <y>270</y>
     <width>148</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Serif</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP_BS" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>410</y>
     <width>210</width>
     <height>140</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Sans Serif</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>BS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_KP_CLS" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>410</y>
     <width>210</width>
     <height>140</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Sans Serif</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>CLS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOk" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>450</x>
     <y>410</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Sans Serif</family>
     <pointsize>35</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>&amp;Enter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MSG" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>480</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>Waiting the Omni's Connection</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_PIN" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>520</x>
     <y>50</y>
     <width>240</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Arial Black</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
